posthog/test/test_organization_model.py

```diff
@@ -28,6 +27,27 @@ def test_organization_active_invites(self):
         self.assertEqual(self.organization.invites.count(), 2)
         self.assertEqual(self.organization.active_invites.count(), 1)
 
+    def test_plugins_are_preinstalled_on_self_hosted(self):
+        with self.settings(TEST=False, MULTI_TENANCY=False):
+            new_org, _, _ = Organization.objects.bootstrap(
+                self.user, plugins_access_level=Organization.PluginsAccessLevel.INSTALL
+            )
+
+        self.assertEqual(
+            Plugin.objects.filter(organization=new_org, is_preinstalled=True).count(),
+            len(settings.PLUGINS_PREINSTALLED_URLS),
```

Comment:
If I read this correctly, this is going to be hitting the Github API a lot. It has a 60 requests per 60 minutes limit. This might cause problems, including our favourite flaky tests. This might be fine in github actions, but not locally and for other users.

Can we stub it somehow? There are already some plugin ZIP files in the shape of a string in `plugin_archives.py` and `mock.py`. Perhaps something can be recycled?

Reply:
Yes, that is amazingly useful, thanks for the pointer. This feature is only enabled in select tests, so low GH load, but even then I refactored them to only use mocks, so all is well